x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/index.tsx

```diff
@@ -176,16 +132,11 @@ const ConfigureCasesComponent: React.FC<ConfigureCasesComponentProps> = ({ userC
     const unsavedChanges = difference(Object.values(currentConfiguration), [
       connectorId,
       closureType,
+      connectorName,
```

Comment:
Only `connectorId` is needed. If we put the connector's name the user will see wrong number of changes (one extra) given that when she changes the connector both of the id and the name is changed at the same time.

Reply:
currentConfiguration contains `connectorName` as well so we don't see one extra